### DISCUSSION TITLE
How do i proceed from a given time lapse?

### DISCUSSION THREAD

**Discussion by HSARG** [openai/whisper#1882] [Q&A]
Im currently using linux command line and transcribing with cpu only (the software for amd gpu isnt robust enough yet). When im transcribing particularly long videos, its somewhat common for the program to die before completting everything. I presume it just runs out of memory or something.

Anyway, its relativally simple for me to copy the transcribed section and save it on a notepad, but i coulndt find any command to resume the transcription from the exact moment where the program died on and foward. I tried to find a solution on the --help section and even here, but didnt found any command for it, only some workarounds. What im missing something?

**Comment by glangford** [comment]
The best workaround is to use ffmpeg, as described here

* https://github.com/openai/whisper/discussions/135#discussioncomment-3731104

**Reply by HSARG** [reply]
Is that so? I thought there would be a more elegant way to do this, but if this is the best solution i guess i can only thank you.

**Reply by glangford** [reply]
Unfortunately there isn't an option within whisper to do this today. I agree it would be nice to have something more elegant.

**Comment by ryanheise** [comment]
You can try #1838 which adds an option to Whisper to specify clip regions. In its simplest usage, you can run whisper with the option `--clip_timestamps STARTSEC` and it will start transcribing from timestamp `STARTSEC` in seconds.
